LinkedList.java

```diff
@@ -4,7 +4,7 @@
 // a Singly Linked List 
 public class LinkedList { 
   
-    Node head; // head of list 
+    static Node head; // head of list 

```

Comment:
Could you please state the reason for head being static here?

Reply:
Oh! There is no reason as such. I was getting warnings from Eclipse regarding this, hence I changed. 
Making it static did not affect my solution as such.

Reply:
Would it work if you try to make more than one Linked Lists using this class ?